### DISCUSSION TITLE
Migration question CC2531 -> CC2652 and Homeassistant

### DISCUSSION THREAD

**Discussion by jimmyeao** [Koenkk/zigbee2mqtt#6647] [General]
I know I have to re-pair all my devices when I move from one coordinator to another, my question is this:
Will it update all my Homeassistant Entities, or will it match the devices to the existing DB entries is Zigbee2mqtt?

**Comment by Koenkk** [comment]
This shouldn't be needed when using the following steps:
- Stop zigbee2mqtt
- Set `homeassistant: false` in `configuration.yaml`
- Change `pan_id` (3rd last point of https://www.zigbee2mqtt.io/information/supported_adapters.html)
- Start zigbee2mqtt
- Pair devices
- Stop zigbee2mqtt
- Set homeassistant: true
- Start zigbee2mqtt

**Reply by jimmyeao** [reply]
Awesome, thanks - I’ll be doing this at the weekend

**Reply by jimmyeao** [reply]
Just done this, worked pretty much flawlessly, only caveat was I had to delete my CC2531 routers and readd them from scratch - re-pairing them initially they showed up as unsupported devices. Delete and re-add, they are now showing fine. :)
